elfpy/markets/hyperdrive.py

```diff
@@ -463,13 +550,8 @@ def open_long(
         tuple[MarketDeltas, wallet.Wallet]
             The deltas that should be applied to the market and agent
         """
-        if trade_amount > self.market_state.bond_reserves:
```

Comment:
why are you removing this? I think we need it for the open long tests

Reply:
i think i may have done this during a rebase.  i'll it back in.